CHANGELOG.md

```diff
@@ -12,8 +12,12 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## Unreleased
 
+- 🎁 Live matching with VAST works again!
+  [#156](https://github.com/tenzir/threatbus/pull/156)
```

Comment:
Shouldn't this go into `apps/vast/CHANGELOG.md`?

Personally I think it's bad UX that we have multiple changelog files for Threat Bus, but that should be a separate discussion.

Reply:
Right, changed.